data_test.json

```diff
@@ -1 +1 @@
-{"name":"Ferko","latestWeight":80.0,"latestHeight":182.0,"records":[{"id":1,"date":"2022-04-03","weight":80.0,"height":185.0,"arm":20.0,"neck":50.0,"waist":35.0,"hip":103.0,"fat":40.0,"muscle":35.0,"water":15.0,"bones":10.0},{"id":2,"date":"2022-04-04","weight":79.0,"height":185.0,"arm":20.0,"neck":50.0,"waist":35.0,"hip":103.0,"fat":40.0,"muscle":35.0,"water":15.0,"bones":10.0},{"id":3,"date":"2022-04-05","weight":81.0,"height":185.0,"arm":20.0,"neck":50.0,"waist":35.0,"hip":103.0,"fat":40.0,"muscle":35.0,"water":15.0,"bones":10.0},{"id":4,"date":"2022-04-06","weight":1.0,"height":185.0,"arm":3.0,"neck":4.0,"waist":5.0,"hip":6.0,"fat":1.5,"muscle":1.5,"water":1.5,"bones":1.5},{"id":5,"date":"2022-04-07","weight":80.0,"height":185.0,"arm":30.0,"neck":30.0,"waist":15.0,"hip":105.0,"fat":1.5,"muscle":1.5,"water":1.5,"bones":1.5},{"id":1,"date":"2022-04-13","weight":100.0,"height":190.0,"arm":30.0,"neck":41.0,"waist":90.0,"hip":90.0,"fat":10.0,"muscle":10.0,"water":10.0,"bones":10.0},{"id":1,"date":"2022-04-13","weight":100.0,"height":190.0,"arm":30.0,"neck":41.0,"waist":90.0,"hip":90.0,"fat":10.0,"muscle":10.0,"water":10.0,"bones":10.0}]}
```

Comment:
Toto musí zostať tak ako to bolo pôvodne. Ten súbor sa používa pri testoch databázovej vrstvy. Ak potrebuješ vlastný JSON tak si ho vytvor pod iným názvom.